### ISSUE TITLE
when I import a c++ head-ony library with a custom pod file, I get an error that file not found with <angled> include; use "quotes" instead

### ISSUE THREAD

**Issue by J844259756** [CocoaPods/CocoaPods#10037]
* [ ] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

ℹ Please replace these two lines with what you did.  
pod lib lint --use-libraries --allow-warnings

## What happened instead?

ℹ Please replace these two lines with of what happened instead.  
- ERROR | [iOS] xcodebuild:  /Users/apple/hellos/ios/Classes/TestWeb.hpp:12:10: error: 'websocketpp/client.hpp' file not found with <angled> include; use "quotes" instead

error: 'websocketpp/roles/client_endpoint.hpp' file not found
    #include <websocketpp/roles/client_endpoint.hpp>

## CocoaPods Environment

ℹ Please replace these two lines with the output of `pod env`.
### Stack

```
   CocoaPods : 1.9.3
        Ruby : ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]
    RubyGems : 3.0.3
        Host : Mac OS X 10.15.4 (19E287)
       Xcode : 11.6 (11E708)
         Git : git version 2.24.3 (Apple Git-128)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib
Repositories : facebookarchive - git - https://github.com/facebookarchive/SocketRocket @ 0a023409b138039d1ab5ac5431fe61c5f334e86b

               master - git - https://github.com/CocoaPods/Specs.git @ 436ba3cafe7d0b0688bb75888e6627032af06f4e

               trunk - CDN - https://cdn.cocoapods.org/
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.4.1
cocoapods-try         : 1.1.0
```

## Project that demonstrates the issue

ℹ Please link to a project we can download that reproduces the issue.
https://github.com/J844259756/WebSocketPPTest.git
the xcode.xcworkspace file is in the path hellos/example/ios/Runner.xcworkspace

**Comment by paulb777** [comment]
Might be addressed in 1.10. See #9902

**Comment by J844259756** [comment]
The solution is invalid ，my xcode is 11.6 and CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER default setting is no

**Comment by J844259756** [comment]
> Might be addressed in 1.10. See #9902
The solution is invalid ，my xcode is 11.6 and CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER default setting is no
